docs/reduction-in-locks.md

```diff
@@ -0,0 +1,107 @@
+# Reduced backup lock time
+
+--8<--- "pro-build-announcement.md"
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+[Percona XtraBackup 8.4.0-2](./release-notes/8.4.0-2.md) adds the [`--lock-ddl=REDUCED`](#how-the---lock-ddlreduced-option-works) option to reduce the time the server remains locked by `xtrabackup` during full and incremental backups. Now, you can execute `Data Definition Language` (DDL) operations while the backup is in progress.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option features are as follows:
```

Comment:
Rewrite the unordered list using a verb as the first word in each item. 

Acquire and Release the Backup Lock Quickly: The backup process minimizes the time it holds the backup lock, allowing for concurrent DDL operations.
Track Changes with Redo Logs: Redo logs record all file-level changes, ensuring data consistency during the backup process.
Handle DDL Operations: The backup process generates metadata files to account for any DDL operations that occur while the backup is in progress.
Ensure Consistency: The --prepare step processes these metadata files and uses redo and undo logs to create a consistent database state.

Reply:
done